kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateList.js

```diff
@@ -318,37 +390,40 @@ define([
                     });
                 })
                 .then(() => {
-                    renderTable(container, jobs.length);
-
-                    jobs.forEach((jobState) => {
-                        startStatusListener(jobState.job_id);
-                        startJobDoesNotExistListener(jobState.job_id);
-                        // populate the job params
-                        startParamsListener(jobState.job_id);
-                        bus.emit('request-job-info', {
-                            jobId: jobState.job_id,
-                        });
+                    renderTable(jobs);
+
+                    container.querySelectorAll('tbody tr.odd, tbody tr.even').forEach((el) => {
```

Comment:
add all the listeners to the table body here, rather than in the JobStateListRow module or elsewhere